awx/main/models/inventory.py

```diff
@@ -2596,6 +2596,7 @@ def get_plugin_env(self, inventory_update, private_data_dir, private_data_files)
 
     def inventory_as_dict(self, inventory_update, private_data_dir):
         ret = super(satellite6, self).inventory_as_dict(inventory_update, private_data_dir)
+        ret['validate_certs'] = False
```

Comment:
I don't think we _need_ this line, and I'd rather leave it to the plugin's default in this case.

Reply:
Okay - I thought that since we're advertising the settings in https://github.com/ansible-collections/community.general/blob/master/scripts/inventory/foreman.ini#L124, that it made sense to pin `validate_certs` to the static value in that file (in case things _do_ change in the plugin).

Then again, I can also see it making sense to follow the plug-in's behavior (and am happy to go either way).

Feel like we're stuck in this weird limbo right now where we want to support backwards compatiblity, but at some point it seems like we'll want to move on and adopt what's in the plugin without managing it / locking it down too much.